frameworklauncher/src/main/java/com/microsoft/frameworklauncher/utils/HadoopUtils.java

```diff
@@ -127,7 +127,7 @@ private static FileStatus getFileStatusInHdfsInternal(String hdfsPath) throws Ex
       LOGGER.logInfo("[hadoop fs -stat %%Y %s]", hdfsPath);
       FileStatus fileStatus = fs.getFileStatus(new Path(hdfsPath));
       return fileStatus;
-    } catch (PathNotFoundException e) {
+    } catch (PathNotFoundException | FileNotFoundException e) {
       throw new NonTransientException("Path does not exist", e);
```

Comment:
>Path does not exist [](start = 39, length = 19)

"hdfsPath"  Path or File Node Found in the message ?

Reply:
Internal Exception itself contains it

---
In reply to: [159354625](https://github.com/Microsoft/pai/pull/156#discussion_r159354625) [](ancestors = 159354625)